Landing page updates and corresponding skill.md (#134)
charmed-hpc/docs: howto/cleanup/index.md

# Clean up resources

> Tear down a Charmed HPC deployment and free the underlying cloud resources when they are no longer needed.

Reviewer: [nit]: "required" instead of "needed" since both Slurm and the underlying cloud resources are mandatory for a functional Charmed HPC cluster. I also think we can be more brief here since the cloud resources are part of a Charmed HPC deployment:

```suggestion
Tear down a Charmed HPC deployment when it is no longer required.
```
